packages/frontend/src/components/table/DurationCell.tsx

```diff
@@ -1,7 +1,10 @@
 import { pluralize } from '@l2beat/shared-pure'
 import React from 'react'
 
-export function DurationCell(props: { durationInSeconds: number }) {
+export function DurationCell(props: {
+  durationInSeconds: number
+  grayedOut?: boolean
```

Comment:
btw. instead of this I was thinking about composition and creating GrayedOut component